Dockerfile

```diff
@@ -50,16 +51,19 @@ FROM ghcr.io/rse-ops/clang-ubuntu-20.04:llvm-11.0.0 AS clang11-debug
 ENV GTEST_COLOR=1
 COPY . /home/raja/workspace
 WORKDIR /home/raja/workspace/build
-RUN cmake -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_BUILD_TYPE=Debug .. && \
+RUN . /opt/spack/share/spack/setup-env.sh && \
+    export LD_LIBRARY_PATH=/opt/view/lib:$LD_LIBRARY_PATH && \
+    cmake -DCMAKE_CXX_COMPILER=clang++ -DENABLE_OPENMP=On && -DCMAKE_BUILD_TYPE=Debug .. && \
     make -j 6 &&\
     ctest -T test --output-on-failure
 
 FROM ghcr.io/rse-ops/clang-ubuntu-22.04:llvm-13.0.0 AS clang13
 ENV GTEST_COLOR=1
 COPY . /home/raja/workspace
 WORKDIR /home/raja/workspace/build
-RUN . /opt/spack/share/spack/setup-env.sh && spack load llvm && \
-    cmake -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_BUILD_TYPE=Release -DENABLE_OPENMP=On .. && \
+RUN . /opt/spack/share/spack/setup-env.sh && \
+    export LD_LIBRARY_PATH=/opt/view/lib:$LD_LIBRARY_PATH && \
+    cmake -DCMAKE_CXX_COMPILER=clang++ -DENABLE_OPENMP=On && -DCMAKE_BUILD_TYPE=Release .. && \
```

Comment:
```suggestion
    cmake -DCMAKE_CXX_COMPILER=clang++ -DENABLE_OPENMP=On -DCMAKE_BUILD_TYPE=Release .. && \
```

Reply:
Thanks Mike. cut-and-paste error. I pushed a fix.